app.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const checkApi = require("./controllers/checkController");
+const app = express();
+
+app.use(express.json());
```

Comment:
Please only submit code relevant to the ticket you're working on. Unless you're working on a post or patch request, `app.use(express.json())` is outside of the scope of this ticket.